tests/test_commands.py

```diff
@@ -70,12 +71,14 @@ def test_parse_command_merge():
     assert len(cmds) == 1
     assert cmds[0].name == "merge"
     assert cmds[0].bumpversion_mode == "patch"
-    cmds = list(parse_commands("/ocabot merge"))
+    cmds = list(parse_commands("/ocabot merge nobump"))
     assert len(cmds) == 1
     assert cmds[0].name == "merge"
-    assert cmds[0].bumpversion_mode is None
+    assert cmds[0].bumpversion_mode == "nobump"
+    with pytest.raises(RequiredOptionError):
+        list(parse_commands("/ocabot merge"))
     with pytest.raises(InvalidOptionsError):
-        list(parse_commands("/ocabot merge brol"))
+        list(parse_commands("/ocabot merge nobump BAD_OPTION"))
```

Comment:
why changing this line?

Reply:
- I saw that "BAD_OPTION" was more lisible that is was wrong. But, not a big deal, I restored ``brol``.
- I added ``nobump`` because it is now required

Reply:
> I thought we could see better that BAD_OPTION was a bad option than brol ;-)

Arguably, yes, no problem with that ;)

> ocabot merge brol

But I mean, "merge brol" was an InvalidOptionError and should still be, so let's leave that test as it is, while adding a new one for "nobump BAD_OPTION".

Reply:
> But I mean, "merge brol" was an InvalidOptionError and should still be, so let's leave that test as it is, while adding a new one for "nobump BAD_OPTION".

In fact no. with this refactoring, \ocabot merge bro1 is an ``InvalidOptionError`` AND a ``RequiredOptionError`` errors and it depends on the order of the check in the function.
As I refactored a little the function (see below) to let the possibility to add other arguments, now, the function check first if all the required arguments are present, and then, if there are remaining argument(s), raise an ``InvalidOptionError`` error.